tools/install-powershell.ps1

```diff
@@ -99,10 +113,17 @@ try {
 
         $package = Find-Package -Source powershell-core-daily -AllowPrereleaseVersions -Name $packageName
         Write-Verbose "Daily package found. Name: $packageName; Version: $($package.Version)" -Verbose
+
         Install-Package -InputObject $package -Destination $tempDir -ExcludeVersion > $null
+        Remove-Destination $Destination
 
         $contentPath = [System.IO.Path]::Combine($tempDir, $packageName, "content")
-        Copy-Item -Path $contentPath\* -Destination $Destination -Recurse -Force
+        if (Test-Path $Destination) {
+            Write-Verbose "Copying files" -Verbose
```

Comment:
I think this log message should be moved before `if (Test-Path $Destination)` since both `copy-item` and `move-item` would take time. Maybe change the message to `Deploying files ...`

Reply:
Move-Item is just updating a single directory pointer so shouldn't take any time